### ISSUE TITLE
Use helper objects to speed up customization of the activities (inspector NG)

### ISSUE THREAD

**Issue by jberends** [KE-works/pykechain#110]
some inspiration

```python
class SuperGrid(object):
    parent = None
    model = None
    
    def __init__(self, parent, model):
        self.model = model
        self.parent = parent
    
    def render(self):
        return {'xtype':'SuperGrid', 'parent': self.parent, 'model':self.model}
    
class Custimisation(object):
    components_list = []
    
    def add_component(self, component):
        self.components_list.append(component)
        
    def render(self):
        return json.dumps({'components': self.components_list}, indent=4)
```

**Comment by jberends** [comment]
see #117

**Comment by jberends** [comment]
released in v1.9.0
